### ISSUE TITLE
How do we Identify Accepted Organizations?

### ISSUE THREAD

**Issue by Gbahdeyboh** [google/season-of-docs#230]
Hi, I am an aspiring GSoD'21 intern.
I have gone through the list of Interested organizations and saw a few that I would really love to work with. 
However, after reaching out to them, I get a response that they were not accepted for this year's GSoD. I've gotten that same response from 2 organizations I reached out to, and a friend got the same response from another organization. 

I previously thought organizations on this list were accepted organizations, but apparently they aren't. How do we identify organizations that were accepted for this year's GSoD?

**Comment by mohzulfikar** [comment]
+1 on this too

**Comment by emckean** [comment]
Hi folks! You can see the accepted orgs for 2021 on the [official Season of Docs site](https://developers.google.com/season-of-docs/docs/participants)

I'll update the repo to add this link! Thanks for pointing this out!

**Comment by Gbahdeyboh** [comment]
Hi @mohzulfikar.

I eventually found it [here](https://developers.google.com/season-of-docs/docs/participants).

Although I might have missed it, I don't think the link visible anywhere(GitHub or GSoCs website) even though I checked severally. On the website, it is on the "results" tab. That was a bit misleading as the name does not give context to the information it contains IMHO. Perhaps it could be updated somewhere where it's more visible?
